src/MainWindow.cpp

```diff
@@ -352,6 +361,13 @@ void MainWindow::showTreemapView()
 	_ui->treemapView->disable();
 }
 
+void MainWindow::treemapAsSidePanel()
```

Comment:
2 empty lines between functions of the same class is the practised standard all over the code. Please observe those standards.